tests/di/BindingsHtmlTest.php

```
@@ -0,0 +1,139 @@
+            rmdir($this->classDir);
+        }
+    }
+
+    public function testEmbedsTheMarkdownVerbatimAndShipsTheDecoration(): void
+    {
+        [$stdout, , $exit] = $this->runTool([$this->md]);
+
+        $this->assertSame(0, $exit);
+        // the <pre> data island unescapes back to the exact markdown
+        preg_match('#<pre id="src">(.*)</pre>#s', $stdout, $matches);
+        $recovered = html_entity_decode($matches[1], ENT_QUOTES, 'UTF-8');
```

PHPStan (2.2.5)

[error] Offset 1 might not exist on array{}|array{non-falsy-string, string}.


(offsetAccess.notFound)